nixos/modules/programs/localsend.nix

```diff
@@ -0,0 +1,27 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+with lib; let
```

Comment:
Maybe you could try to avoid file-wide `with lib` statements? Either inheriting the functions you are using, or accessing them directly using `lib.<fun>` would be better.

Reply:
Changed `with lib` to `lib.<fun>`.